core/msg.c

```diff
@@ -370,6 +370,25 @@ static int _msg_receive(msg_t *m, int block)
     DEBUG("This should have never been reached!\n");
 }
 
+int msg_avail(void)
+{
+    unsigned state = disableIRQ();
+    DEBUG("msg_available: %" PRIkernel_pid ": msg_available.\n",
+          sched_active_thread->pid);
+
+    tcb_t *me = (tcb_t*) sched_threads[sched_active_pid];
```

Comment:
@gebart So for this PR I keep using `sched_threads[sched_active_pid]` to be on the save side?